pkg/ebpf/c/maps.h

```diff
@@ -106,6 +120,7 @@ BPF_PROG_ARRAY(sys_enter_submit_tail, MAX_EVENT_ID);               // store prog
 BPF_PROG_ARRAY(sys_exit_submit_tail, MAX_EVENT_ID);                // store program for submitting syscalls from sys_exit
 BPF_PROG_ARRAY(sys_enter_init_tail, MAX_EVENT_ID);                 // store program for performing syscall tracking logic in sys_enter
 BPF_PROG_ARRAY(sys_exit_init_tail, MAX_EVENT_ID);                  // store program for performing syscall tracking logic in sys_exits
+BPF_PROG_ARRAY(hidden_kernel_mod_prog_arr, MAX_TAIL_CALL);         // store program for hidden kernel module logic
```

Comment:
is this map required? why not using prog_array map?

Reply:
Not required - switched to prog_array